lib/Runtime/Library/CMakeLists.txt

```diff
@@ -114,6 +114,13 @@ set(CRLIB_SOURCE_CODES
     TypedArray.cpp
     TypedArrayIndexEnumerator.cpp
     UriHelper.cpp
+    WasmLibrary.cpp
```

Comment:
I would really appreciate if you can add these sources conditionally. 

```
if (CC_TARGETS_AMD64)
    set(CRLIB_SOURCE_CODES ${CRLIB_SOURCE_CODES}
        WasmLibrary.cpp
        WebAssembly.cpp
        .... and others
       )
endif()
```

..and an `xplat-todo` note `enable this for other arch(s)` same applies to other CMake etc. changes.